### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gitbock**
path to config file added

### CODE REVIEW

File: addons/io/org.openhab.io.hueemulation/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Hue Emulation exposes openHAB items as Hue devices to other Hue HTTP API compati
 * Pairing (security) can be enabled/disabled in real time using the configuration service (under services in the PaperUI for example)  
 
 ## Configuration:
-
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.hueemulation/README.md]: you must not remove the empty line

File: addons/io/org.openhab.io.hueemulation/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Hue Emulation exposes openHAB items as Hue devices to other Hue HTTP API compati
...
+in "userdata/config/org/openhab/hueemulation.config"
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.hueemulation/README.md]: this does not sound like  sentence to me - should it be combined with the next line?

Comment (gitbock) [inline addons/io/org.openhab.io.hueemulation/README.md]: this was only a suggestion to mention where the config file is located. Did not find it at once.

Comment (kaikreuzer) [inline addons/io/org.openhab.io.hueemulation/README.md]: Right, thanks for that. But the docs should nonetheless contain proper sentences.
And afaik, the correct place for such a configuration is conf/services/runtime.cfg - I guess this should be covered in general by the main documentation through https://github.com/openhab/openhab-docs/issues/309.

**Pull request closed without merging**